server/core/rest/user.py

```diff
@@ -27,3 +30,32 @@
 @user_only
 def get_user(user_id, user_email):
     return jsonify(user_id)
+
+
+@user_api.route("/enrollments", methods=["GET", "POST", "DELETE"])
+@user_only
+def enrollments(user_id, user_email):
+    data = request.json
+    e_id = request.args.get('enrollment_id')
+    method = request.method
+
+    if method == 'GET':
+        if e_id is None:
+            query = Enrollments.select().dicts()
+            enrollments = [e for e in query]
+            result = enrollments
+        else:
+            result = Enrollments.get(id=e_id)
+    elif method == 'POST':
+        study_id = data.get('study_id')
+        result = Enrollments.create(user=user_id, study=study_id)
+        result = model_to_dict(result)
+        del result['study']
```

Comment:
From what I remember, I think I saw a list of users being returned in the `study` object in this response, so I deleted that just in case (didn't want the user to be able to look at other users' data). Perhaps Pewee is more clever than that, but I was aware I didn't have enough time to investigate that.